libbeat/_meta/config.reference.yml.tmpl

```diff
@@ -1081,6 +1081,23 @@ setup.kibana:
   # Configure curve types for ECDHE-based cipher suites
   #ssl.curve_types: []
 
+  # Authentication type to use with Kerberos. Available options: keytab, password.
+  #kerberos.auth_type: password
+
+  # Path to the keytab file. It is used when auth_type is set to keytab.
+  #kerberos.keytab: /etc/elastic.keytab
+
+  # Path to the Kerberos configuration.
+  #kerberos.config_path: /etc/krb5.conf
+
+  # Name of the Kerberos user.
+  #kerberos.username: elastic
+
+  # Password of the Kerberos user. It is used when auth_type is set to password.
+  #kerberos.password: changeme
+
+  # Kerberos realm.
+  #kerberos.realm: ELASTIC
```

Comment:
These are all under `setup.kibana`. Is that correct or did you mean to put them under `output.elasticsearch` (or both)?

Reply:
:facepalm: :facepalm: :facepalm:

Reply:
Fixed. https://images-cdn.9gag.com/photo/6324230_700b.jpg :D :D :D